src/app/Models/FinancialOperation.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class FinancialOperation extends Model
+{
+    use HasFactory;
+
+    /**
+     * Indicates if the model should be timestamped.
+     *
+     * @var bool
+     */
+    public $timestamps = false;
+
+    /**
+     * Array of attributes excluded from mass assignation.
+     *
+     * @var string[]
+     */
+    protected $guarded = ['id'];
+
```

Comment:
Add this to enable eager loading of operationType (and avoid N+1 select problem):
```suggestion
protected $with = ['operationType'];
```